javascript/javascript2/week3/fastest-presser.js

```diff
@@ -0,0 +1,72 @@
+/*
+When the button is clicked, get the value of the input.
+This value will be the amount of time the game should run.
+*/
+
+const gameButton = document.querySelector("#start"); //start game button
+const input = document.querySelector("input");
+let sCount = 0;
+let lCount = 0;
+
+const resultTag = document.getElementById("game-result"); //game result
+resultTag.innerHTML = "";
+const gameResult = () => {
+  if (sCount == 0 && lCount == 0) {
+    resultTag.innerHTML = "No action!!!";
+  }
+  if (sCount == lCount && sCount > 0 && lCount > 0) {
```

Comment:
Many `If ` the conditions are exclusive, try to use `else if`, this will save multiple condition checking.

Reply:
thanks alot 👍